src/index.ts

```diff
@@ -47,6 +48,11 @@ program //
             .option('--appId [applicationId]')
             .argument('[path]', 'path with respect to current working directory that will locate all published files')
             .action(publish)
+    )
+    .addCommand(
+        new Command(list.name) //
```

Comment:
Merge build app cli first into main, then modify the command to lazily load list command